### DISCUSSION TITLE
can send me one TXCH in testnet9? i want to test pools, please

### DISCUSSION THREAD

**Discussion by lpf763827726** [Chia-Network/chia-blockchain#7032] [Support]
I checkout testnet9 branch, i found my TXCH is 0, i can't join my pool test

My disk only can plot the k25, and Estimated network space is 8.6T now, it need too long time to win

Can you send me one TXCH, please, i want to join my pool and keep test

my wallet is: txch1cc2cu0ngvej0j8dlu6gz7ataq07e884802cnu9yes7cqrfvu0mvq0kj2ju

**Comment by loppefaaret** [comment]
some txch is on the way

**Reply by lpf763827726** [reply]
I got it, Thank you very much! :)

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
